pipelines/rj_crm__geolocalizacao_residencia/flow.py

```
@@ -111,6 +111,8 @@ def rj_crm__geolocalizacao_residencia(
 
     # Check if we have addresses to process
     empty_addresses = check_df_emptiness(dataframe=dataframe)
+    if empty_addresses:
+        print(f"Address dataframe empty")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)